### ISSUE TITLE
Rosetta: Should Korean use `text-wrap: balance`?

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-parent-2021#150]
I was testing out something else and noticed that the Korean rosetta site has some headings where words appear cut off. For example, "community" (커뮤니티를) below is a single word, but it's split in half. I assume this is the same issue as #123 in Japanese, where the `text-wrap: balance` is causing issue.

<img width="640" alt="ko-balance" src="https://github.com/user-attachments/assets/a72849e0-5ca3-4411-9eaa-450197e1ab84">

<img width="529" alt="Screenshot 2024-08-09 at 5 45 49 PM" src="https://github.com/user-attachments/assets/71f7a634-3efb-4813-8a90-50db773b1b09">

Is this a problem for Korean readers like it was in Japanese, or is it common/expected that words will wrap? cc @StevenDufresne

**Comment by taggon-dable** [comment]
Yes, this is a problem for Korean readers. Those copies should loook like:

```
워드프레스 커뮤니티를
만나보세요
```

and

```
워드프레스를
만나보세요
```

Can we apply `word-break: keep-all` to the headers? Open [a live demo](https://jsbin.com/vugibusopi/1/edit?html,css,output) to see how it would look.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/39e83038-4fc0-4539-bc7f-18f80a311ec8">

**Comment by taggon** [comment]
Yes, this is a problem for Korean readers. Those copies should loook like:

```
워드프레스 커뮤니티를
만나보세요
```

and

```
워드프레스를
만나보세요
```

Can we apply `word-break: keep-all` to the headers? Open [a live demo](https://jsbin.com/vugibusopi/1/edit?html,css,output) to see how it would look.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/39e83038-4fc0-4539-bc7f-18f80a311ec8">
